packages/application-shell/src/components/navbar/use-initial-forced-menu-state.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import isNil from 'lodash/isNil';
+import { STORAGE_KEYS } from '../../constants';
+
+const useInitialForcedMenuState = () => {
```

Comment:
Nice! Nit: Maybe `useMenuLayoutEffect` to indicate what the hook does. It's not really returning state or anything which the name otherwise suggsts.